addons/l10n_de/models/chart_template.py

```diff
@@ -16,4 +16,37 @@ def _load(self, company):
                 'external_report_layout_id': self.env.ref('l10n_din5008.external_layout_din5008').id,
                 'paperformat_id': self.env.ref('l10n_din5008.paperformat_euro_din').id
             })
+
+            outstanding_receipt = company.account_journal_payment_debit_account_id
+            outstanding_payment = company.account_journal_payment_credit_account_id
+
+            outstanding_receipt['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
+            outstanding_payment['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
+
+        return res
+
+    def _prepare_transfer_account_template(self):
+        res = super(AccountChartTemplate, self)._prepare_transfer_account_template(None)
+        if self in [
+            self.env.ref('l10n_de_skr03.l10n_de_chart_template', raise_if_not_found=False),
+            self.env.ref('l10n_de_skr04.l10n_chart_de_skr04', raise_if_not_found=False)
+        ]:
+            tag_ids = res.get('tag_ids', [])
+            tag_ids.append((4, self.env.ref('l10n_de.tag_de_asset_bs_B_II_4').id))
+            res['tag_ids'] = tag_ids
```

Comment:
Doesn't `+=` work in this case instead of using an ORM command? (no clue)

If the ORM command is absolutely necessary, We should use the `Command.Link(record.id)` format instead of this one

Reply:
It's exactly the same: Command.link just convert to something similar you wrote.
It's just for more clarity and obviousness.

Reply:
feedback summary given by pm:
> Doesn't += work in this case instead of using an ORM command? (no clue)

I thought `res` was a record but it's just a dict and thus `+=` isn't an option